README.md

```diff
@@ -1,5 +1,35 @@
 [![jsbeeb tests](https://github.com/mattgodbolt/jsbeeb/actions/workflows/test-and-deploy.yml/badge.svg)](https://github.com/mattgodbolt/jsbeeb/actions/workflows/test-and-deploy.yml)
 
+
+
+Music 5000 emulation
+--------------------
+
+I've ported over the Beebem code for the Music 5000. This version uses Audio Worklets so 
```

Comment:
Who is the "I" here? :) All this being at the top of the README is confusing - is the plan to remove this?

Reply:
I've undone the update to README.md